speechbrain/pretrained/interfaces.py

```diff
@@ -2221,7 +2221,7 @@ def enhance_batch(self, noisy, lengths=None):
         # Return resynthesized waveforms
         return self.hparams.resynth(torch.expm1(enhanced), noisy)
 
-    def enhance_file(self, filename, output_filename=None):
+    def enhance_file(self, filename, output_filename=None, sample_rate=16000):
```

Comment:
I don't think taking a `sample_rate` parameter here is a good idea. It is confusing because it must be made to match the input sample rate, but that is already known at `load_audio` time. Mismatching the sample rate will only silently produce an output file with an incorrect sample rate.

Reply:
OK, that's fine.

I think the community will fix it at some point.